engines/gob/detection/tables_littlered.h

```diff
@@ -252,7 +252,7 @@
 		"",
 		AD_ENTRY2s("intro.stk",		"0b72992f5d8b5e6e0330572a5753ea25", 256490,
 				   "mod.babayaga",	"43484cde74e0860785f8e19f0bc776d1", 60248),
-		UNK_LANG,
+		EN_ANY,
```

Comment:
The Amiga release is multilanguage (see the [Hall of Light entry](https://amiga.abime.net/games/view/little-red-riding-hood), which is a database of Amiga games), so I don't think it should have the English flag

Reply:
Thank your response, then i change it back to UNK_LNG

Reply:
fixed